.github/workflows/docker-push.yml

```
@@ -0,0 +1,84 @@
+---
+name: 'Push Docker Image'
+on:
+  workflow_dispatch:
+
+    inputs:
+      kontrol_branch:
+        description: "Branch of Kontrol to use to build the Docker image"
+        required: true
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  dockerhub:
+    name: 'Build and Publish Docker Image'
+    runs-on: [self-hosted, linux, normal]
+    steps:
+      - name: 'Check out code'
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.events.inputs.kontrol_branch}}
```

```suggestion
          ref: ${{ github.event.inputs.kontrol_branch}}
```

Should it be `event`?

---

I think so, thanks, fixed in https://github.com/runtimeverification/kontrol/pull/913/commits/12be301560c01120da1d855539c66c99c602d04b.